src/applications/mhv/secure-messaging/tests/e2e/pages/SecureMessagingLandingPage.js

```diff
@@ -36,6 +36,33 @@ class SecureMessagingLandingPage {
       .should('be.visible')
       .and('have.text', `${text}`);
   };
+
+  verifyUnreadMessagesNote = (text = 'unread messages in your inbox') => {
+    cy.get('.vads-u-font-size--h3')
```

Comment:
If possible, change from selecting on styling attribute

Reply:
agree, will ask devs to add more data-testid attributes

Reply:
requested changes done